library/CMakeLists.txt

```diff
@@ -116,10 +116,19 @@ set_target_properties(libf3d PROPERTIES
   POSITION_INDEPENDENT_CODE ON
   OUTPUT_NAME "f3d"
   PDB_NAME "libf3d"
-  VERSION ${F3D_MAJOR_VERSION}.${F3D_MINOR_VERSION}
-  SOVERSION ${F3D_MAJOR_VERSION}
   )
 
+# It can be useful to disable soversion in case the links are duplicated
+# It happens with Python wheels for example
```

Comment:
why would they be duplicated ? how does that look ?

Reply:
We have `libf3d.so`, `libf3d.so.2.1` and `libf3d.so.2.1.0` but no links, they are plain duplicated files. Not sure what happens internally...

Reply:
That would be worth investigating. You see this only with python wheels?

Reply:
Yes of course. Works fine on regular Linux builds.

Reply:
I think it's done on purpose by the "repair" step.  
The wheels are supposed to be self-contained, so dependencies binaries are copied into the wheels, and links are probably converted in the binary in case the link is pointing outside of the wheel.